### ISSUE TITLE
Flat buttons don't look like buttons

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3550]
## Describe the bug

A flat button looks more like a link, and it might not be clear that they are actually buttons.

## How to reproduce

https://clarity.design/documentation/buttons

- View buttons documentation
- View flat buttons 

## Expected behavior

Buttons should be visually consistent.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by gnomeontherun** [comment]
If a flat button is considered invalid or inaccessible, then there are other option button choices available.

**Comment by colinreedmiller** [comment]
The flat button is a pretty common component and is used in many products. Changing it will be visually breaking and will likely be significant for some products. Because of this and its common  presence in numerous UI systems, this should remain low priority and be reviewed by a11y - it appears that this is likely not a significant issue.

**Comment by chlane** [comment]
Closing this. My opinion was that flat buttons would not be visually distinguishable from other text making it difficult for sighted speech recognition users to visually perceive these controls as buttons but I have been assured that these will be styled in such a way that it is visually obvious these are buttons.
